scripts/run-dependencies.sh

```diff
@@ -66,3 +72,7 @@ $here/cli.sh add "Password Reset Test User" passwordtest.user \
     --if-not-exists
 
 $here/cli.sh addRole passwordtest.user user
+
+# Add user to orderly_web
+$here/orderly_web_cli.sh add-users test.user@example.com
+$here/orderly_web_cli.sh grant test.user@example.com */reports.read
```

Comment:
Don't think this is needed because `auth.fine_grained=false`

Reply:
As discussed, will retain this, but turn on auth.fine_grained to mirror real montagu